README.md

```diff
@@ -46,33 +46,31 @@ questions. [Join the Discord](https://discord.gg/cEVEmqCgWG) and say hi! We are
 developers of all skill levels to Fedimint and associated open-source Bitcoin projects. Fedimint touches many different
 areas of Bitcoin development, so there is something for everyone. See below for more information on how to get involved.
 
-## Running your own Fedimint
+## Using Fedimint
 
-The preferred way to run Fedimint is with multiple **guardian servers** working together in a federation. Fedimint is
-built to be **Byzantine Fault Tolerant**, meaning it can continue functioning even if some guardians act maliciously.
-Specifically, a federation with `3m + 1` guardians can tolerate up to m malicious guardians. For example:
-	•	A federation with 4 guardians can handle 1 malicious guardian.
-	•	A federation with 7 guardians can handle 2 malicious guardians.
+To use Fedimint you only need a client application, that will allow interacting with
+Fedimint federations.
 
-You can also run Fedimint in **solo mode** with just one guardian. While this is useful for testing or development, it
-is not recommended for real-world use since it lacks the resilience of a federation with multiple guardians.
+You can pick one of Fedimint-supporting applications:
 
-To support **Lightning payments**, Fedimint requires a **Lightning Gateway**. This is a participant in the federation
-that swaps ecash for sending and receiving Lightning payments. Note that the Lightning Gateway is not a guardian—it’s an
-untrusted economic actor that interacts with the federation.
+* [Fedi](https://www.fedi.xyz/) - for MacOS, Android and Web browsers
+* [Harbor Wallet](https://harbor.cash/) - desktop wallet
+* [Mutiny Wallet](https://www.mutinywallet.com/) - web wallet
```

Comment:
Shut down afaik

Reply:
@elsirion I thought so too, but there's nothing there I can see at the glance that would indicate it.

Reply:
I'm pretty sure when you open the app it tells you, so not a good on-boarding UX.